### PR TITLE
#401. Refactor - remove RawSourceJudgmentParser

### PR DESCRIPTION
**Pull request by madryk**
see #401

### CODE REVIEW

File: saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/constitutionaltribunal/judgment/json/SourceCtJudgment.java [hunk 1]
```diff
@@ -18,16 +15,12 @@
  * @author madryk
  */
 @JsonIgnoreProperties(ignoreUnknown = true)
```
Comment (lukdumi) [inline saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/constitutionaltribunal/judgment/json/SourceCtJudgment.java]: This annotation is present in the super class. Isn't it enough?

File: saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/supremecourt/judgment/json/SourceScJudgment.java [hunk 2]
```diff
@@ -21,38 +15,21 @@
  * @author Łukasz Dumiszewski
  */
 @JsonIgnoreProperties(ignoreUnknown = true)
```
Comment (lukdumi) [inline saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/supremecourt/judgment/json/SourceScJudgment.java]: Same here.

File: saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/constitutionaltribunal/judgment/json/SourceCtJudgment.java (hunk 1 above)
Comment (madryk) [inline saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/constitutionaltribunal/judgment/json/SourceCtJudgment.java]: Yes. You're right.

File: saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/supremecourt/judgment/json/SourceScJudgment.java (hunk 2 above)
Comment (madryk) [inline saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/supremecourt/judgment/json/SourceScJudgment.java]: :+1:

**Pull request closed without merging**